libraries/ms-common/test/test_can.c

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
We should add some kind of check/warning to `hal_test_helpers` and call it in both `test_can` and `test_can_hw` to validate socketcan is running on `vcan0`. I was able to get to the test and it just hung forever before I realized I needed to start socketcan. I spent like 1 hr trying to debug because there was no debug messages or warning of any kind. The only way I realized it was printing the `ioctl` error and then looking at the `/proc` filesystem for devices.

Reply:
Also added socketcan init to test/run-related targets on x86